shrinkwrap.yaml

```diff
@@ -1,267 +0,0 @@
-dependencies:
```

Comment:
I know that pnpm renamed its lockfile, but i don't see the new lockfile in this PR. Just deleting it does not do it.

Reply:
I will commit `package-lock.json` later

Reply:
no, please use pnpm and commit `pnpm-lock.yaml`